.github/workflows/board.yaml

```diff
@@ -0,0 +1,18 @@
+name: Project Board
```

Comment:
Why is this file udpated?

Reply:
File is a duplicate from board.yml I have removed it.